src/devices/cpu/mips/mips3.cpp

```diff
@@ -1713,6 +1891,32 @@ uint64_t mips3_device::get_cop0_reg(int idx)
 	return m_core->cpr[0][idx];
 }
 
+void r4650_device::set_cop0_reg(int idx, uint64_t val)
+{
+	switch (idx)
+	{
+		case COP0_R4650_IBase:
+			m_core->cpr[0][idx] = val;
+			printf("R4650 IBase: %08x%08x\n", (uint32_t)(val >> 32), (uint32_t)val);
+			break;
+		case COP0_R4650_IBound:
+			m_core->cpr[0][idx] = val;
+			printf("R4650 IBound: %08x%08x\n", (uint32_t)(val >> 32), (uint32_t)val);
+			break;
+		case COP0_R4650_DBase:
+			m_core->cpr[0][idx] = val;
+			printf("R4650 DBase: %08x%08x\n", (uint32_t)(val >> 32), (uint32_t)val);
+			break;
+		case COP0_R4650_DBound:
+			m_core->cpr[0][idx] = val;
+			printf("R4650 DBound: %08x%08x\n", (uint32_t)(val >> 32), (uint32_t)val);
+			break;
```

Comment:
Could you please change these to `logerror` or `LOG*`?

Reply:
I just removed the prints entirely and simplified the switch/case.